app/assets/javascripts/listings/templates/listing/_lottery_results.html.slim

```diff
@@ -1,7 +1,7 @@
 h4.t-gamma.t-serif Lottery Results
 p.content-group_day.margin-bottom
   | {{ $ctrl.parent.listing.Lottery_Results_Date | date : 'longDate' }}
-.padding.bg-dust
+.padding.bg-dust ng-if="$ctrl.lotteryComplete()"
```

Comment:
do we still need this if we have the control a level up?

Reply:
for line 11 below:
```
  a.button.primary.expand.no-margin ng-if="$ctrl.showDownloadLotteryResultsButton()" ng-href="{{$ctrl.parent.listing.LotteryResultsURL}}" target="_blank" ng-disabled="$ctrl.loading.lotteryResults" data-event="gtm-download-lottery-results"
```
Should we add in the `$ctrl.loading.lotteryResults` to the ng-if so we only see it if the results are no longer loading and there's still no buckets?